src/filesources/gio/ugiofilesourceutil.pas

```diff
@@ -205,10 +205,10 @@ procedure FillAndCount(Files: TFiles; CountDirs: Boolean; out NewFiles: TFiles;
 function FileExistsMessage(SourceFile: TFile; TargetInfo: PGFileInfo; const TargetName: String): String;
```

Comment:
This unit does not have `uDCUtils` in the uses clause.

Reply:
OK. I'll Add it.